independent-projects/resteasy-reactive/server/runtime/src/main/java/org/jboss/resteasy/reactive/server/spi/ServerHttpRequest.java

```diff
@@ -34,6 +34,8 @@ public interface ServerHttpRequest {
 
     String getQueryParam(String name);
 
+    Map<String, List<String>> getParametersMap();
```

Comment:
I'd prefer we call this `getQueryParamsMap`

Reply:
ok